src/nexttick.js

```diff
@@ -25,14 +21,12 @@ const nextTick = (() => {
     };
 
     return task => {
-      tail = tail.next = { task: task };
+      tail = tail.next = { task };
       channel.port2.postMessage(0);
     };
   }
 
-  return task => {
-    setTimeout(task, 0);
-  };
+  return task => setTimeout(task, 0);
```

Comment:
These arrow functions are block statement because the nextTick is not needed return value for optimization.